test/com/google/javascript/jscomp/CodePrinterTest.java

```diff
@@ -47,6 +48,39 @@ public void testBigInt() {
     assertPrint("-0b110n", "-6n");
     assertPrint("-0o7n", "-7n");
     assertPrint("-0x8n", "-8n");
+    assertPrintSame("1000000000n");
+    assertPrintSame("-10000000000n");
+
+    // Normalize the results to lowercase so the tests do not enfore the output is lowercase but
+    // instead the capitalization is left to the compiler.
+    Function<String, String> normalizeBigInt = s ->
+        s.substring(0, s.length() - 1).toLowerCase() + s.substring(s.length() - 1);
```

Comment:
What's the downside of skipping this normalization in the test cases?

Reply:
My general inclination is to try to keep the testing infrastructure simpler, even if that means we need to update the test cases more frequently.  I think this makes it easier to read and understand the tests.